### PR TITLE
Added electron app

### PR DESCRIPTION
**Pull request by Nick-McGee**
This is just the source code of the electron app, it doesn't have the node modules from electron. Because of this, if you want to run it you have to create an electron app and import the source code into it, as it's not good practice to commit the node modules to the repo.

### CODE REVIEW

**Review by Ansper11 — COMMENTED** [review]
Excellent job, it is very helpful for our project. But can you make a brief tutorial about how to build an electron app?

**Comment by yyht6300a** [comment]
I agree with Andy. Maybe just briefly explain how it works.

File: electronApp/src/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <img src="img/lesson2.png" alt="lesson2" class="lesson-img">
+        <p class="lesson-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur labore provident mollitia, id esse veritatis necessitatibus assumenda amet suscipit! Fugit.</p>
+        <a href="lesson2.html" class="btn btn-play">Play</a>
+      </div>
+
+      <div class="lesson">
+        <h1 class="lesson-h1">Lesson 3</h1>
+        <img src="img/lesson3.png" alt="lesson2" class="lesson-img">
+        <p class="lesson-desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur labore provident mollitia, id esse veritatis necessitatibus assumenda amet suscipit! Fugit.</p>
+        <a href="lesson2.html" class="btn btn-play">Play</a>
```
Comment (Ansper11) [inline electronApp/src/index.html:35]: This page looks great

File: electronApp/src/js/lesson1.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+fs.readFile('src/lessonTemplates/lesson1.py', (err, data) => { 
+  if (err) 
+    throw err; 
+  
+  lessonText = data.toString();
+}) 
+
+// Run lesson code
+var pyShell = pythonShell.PythonShell;
+var outputText = 'error'
```
Comment (Ansper11) [inline electronApp/src/js/lesson1.js:23]: Are we able to create multiple cells on this page and run each cell independently? like Jupiter notebook

Comment (Nick-McGee) [reply electronApp/src/js/lesson1.js:23]: I can look into that

**Pull request merged**